internal/apis/kfd/v1alpha2/eks/create/kubernetes.go

```diff
@@ -23,24 +24,34 @@ import (
 	"github.com/sighupio/furyctl/configs"
 	"github.com/sighupio/furyctl/internal/cluster"
 	"github.com/sighupio/furyctl/internal/template"
+	"github.com/sighupio/furyctl/internal/tool/awscli"
 	"github.com/sighupio/furyctl/internal/tool/terraform"
 	bytesx "github.com/sighupio/furyctl/internal/x/bytes"
 	execx "github.com/sighupio/furyctl/internal/x/exec"
 	iox "github.com/sighupio/furyctl/internal/x/io"
 	kubex "github.com/sighupio/furyctl/internal/x/kube"
+	netx "github.com/sighupio/furyctl/internal/x/net"
 )
 
 var (
-	errKubeconfigFromLogs = errors.New("can't get kubeconfig from logs")
-	errPvtSubnetNotFound  = errors.New("private_subnets not found in infra output")
-	errPvtSubnetFromOut   = errors.New("cannot read private_subnets from infrastructure's output.json")
-	errVpcCIDRFromOut     = errors.New("cannot read vpc_cidr_block from infrastructure's output.json")
-	errVpcCIDRNotFound    = errors.New("vpc_cidr_block not found in infra output")
-	errVpcIDNotFound      = errors.New("vpc id not found: you forgot to specify one or the infrastructure phase failed")
+	errKubeconfigFromLogs  = errors.New("can't get kubeconfig from logs")
+	errPvtSubnetNotFound   = errors.New("private_subnets not found in infra output")
+	errPvtSubnetFromOut    = errors.New("cannot read private_subnets from infrastructure's output.json")
+	errVpcCIDRFromOut      = errors.New("cannot read vpc_cidr_block from infrastructure's output.json")
+	errVpcCIDRNotFound     = errors.New("vpc_cidr_block not found in infra output")
+	errVpcIDNotFound       = errors.New("vpc id not found: you forgot to specify one or the infrastructure phase failed")
+	errParsingCIDR         = errors.New("error parsing CIDR")
+	errResolvingDNS        = errors.New("error resolving DNS record")
+	errVpcIDNotProvided    = errors.New("vpc_id not provided")
```

Comment:
I'd recommend using the field name as in the configuration file (`vpcId`), so the user knows what to search for, or a descriptive text like `VPC ID`:
```suggestion
	errVpcIDNotProvided    = errors.New("vpcId not provided")
```